lib/src/base-styles/_nx-radio-checkbox.scss

```diff
@@ -32,12 +32,18 @@
   }
 
   &:focus-within {
-    &::after {
-      border: 1px solid var(--nx-color-interactive-border-focus);
-      box-shadow: var(--nx-box-shadow-focus);
-      content: '';
-      display: block;
-      position: absolute;
+    // For radios, .nx-radio-checkbox__control is added to an <svg>
+    // Since <svg> does not support pseudoelements, it is instead added to
+    // an additional <span> with the class nx-radio__focus
+    .nx-radio-checkbox__control, .nx-radio__focus {
```

Comment:
This combination of classes results in the attributes being applied to both the radio's SVG and `span`. I think that you need to replace `nx-radio-checkbox__control` with `nx-checkbox__box`

Reply:
> This combination of classes results in the attributes being applied to both the radio's SVG and `span`. I think that you need to replace `nx-radio-checkbox__control` with `nx-checkbox__box`

This has been fixed.
ef22d2ecd2cc3eb0b8b7e1804cf7a32fcf4a76e4